### ISSUE TITLE
panic handler behaviour (Invoke GDBStub) link error

### ISSUE THREAD

**Issue by bvernoux** [espressif/esp-idf#68]
I have updated esp-idf to latest commit (from today nov 1, 2016 but IIRC I had this problems 3 days ago too) and when I build esp-idf/examples/02_blink with make menuconfig => Component config => ESP32-specific config => panic handler behaviour (Invoke GDBStub)
I have a link error (panic.c => esp_gdbstub_panic_handler which is not referenced from )
```
make[1]: Leaving directory `/esp-idf/examples/02_blink/build/main'
LD blink.elf
/esp-idf/examples/02_blink/build/esp32/libesp32.a(panic.o):(.literal.commonErrorHandler+0x20): undefined reference to `esp_gdbstub_panic_handler'
/esp-idf/examples/02_blink/build/esp32/libesp32.a(panic.o): In function `commonErrorHandler':
/esp-idf/components/esp32/./panic.c:271: undefined reference to `esp_gdbstub_panic_handler'
collect2: error: ld returned 1 exit status
make: *** [/esp-idf/examples/02_blink/build/blink.elf] Error 1
```
I have found esp-idf/components/esp32/gdbstub.c => gdbstubPanicHandler function which I suspect is a good candidate to fix this issue maybe it shall be renamed esp_gdbstub_panic_handler ?
* I have renamed  esp-idf/components/esp32/gdbstub.c => gdbstubPanicHandler to esp_gdbstub_panic_handler and the problem seems solved anyway I think the whole file need name refactor with esp_ ...

**Comment by projectgus** [comment]
Hi @bvernoux,

Thanks for reporting this. The fix you have made is the correct one. @Spritetm has a patch for this in our internal review queue, it should be merged very soon.

Angus

**Comment by projectgus** [comment]
Should be fixed now, as of 8c1d1e1

Thanks,

Angus
